### DISCUSSION TITLE
Not able to access Keycloak Admin / Management Interface

### DISCUSSION THREAD

**Discussion by mozdi** [keycloak/keycloak#32417] [Q&A]
After changing form keycloak 24.0.4 to keycloak 25.0.4 (installed directly on Linux), the admin interface is not reachable anymore.
I´ve changed the configuration file to meet the Hostname v2 settings but when connect I only get "Keycloak Management Interface" without additional path and "Resource not found" for /admin.
For path "/health" I get the expected result:
> {
>     "status": "UP",
>     "checks": [
>     ]
> }

Here´s the relevant part of my configuration file:

```
# Observability
# If the server should expose healthcheck endpoints.
health-enabled=true

# If the server should expose metrics endpoints.
# metrics-enabled=true

# HTTP
# The file path to a server certificate or certificate chain in PEM format.
https-certificate-file=${kc.home.dir}/conf/server.crt.pem

# The file path to a private key in PEM format.
https-certificate-key-file=${kc.home.dir}/conf/server.key.pem

# Hostname for the Keycloak server.
hostname=https://auth._domain_.de
hostname-admin=https://keycloak._internal-domain_.de/admin

#https-management-certificate-file=server.crt.pem
#https-management-certificate-key-file==server.key.pem

#Run only on IPv4
java.net.preferIPv4Stack=true
```

Are the configured options correct, or do I need any additional option to aktivate the management interface?
Do I need to set the https-management-certificate-file / key-file options or does the management use the samle certificates as the user-part uses?

Thanks for your help in advance!

**Comment by mshri628** [comment]
@mozdi  are you facing something similar https://keycloak.discourse.group/t/admin-console-not-accessible-with-keycloak-version-25-0-0-and-hostname-v2-configurations/27689 ?
